packages/react/src/context/slash-id-context.tsx

```diff
@@ -125,14 +125,7 @@ export const SlashIDProvider: React.FC<SlashIDProviderProps> = ({
     setUser(undefined);
     // we need to set the oid back to the root on log out
     setOid(initialOid);
-    // we can't reinitialize the SDK here, as it picks up the old token and creates a new user immediately
-    sidRef.current = new SlashID({
```

Comment:
The SDK instance is created with an `oid` - when we log out, we won't reset this OID, but when we switch orgs we create a new SDK instance with the new OID. Am I missing something, do we have another way of clearing this value now?

Reply:
@jake-slashid @kaspermroz looking in more detail, passing `oid` to the SlashID constructor is inconsequential - that value is not being used. I remember adding it there as a stepping stone to the next major version. We still pass the oid as a mandatory parameter to `sdk.id()` every time we log in.